src/components/v2/shared/NftPostPayModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { t } from '@lingui/macro'
+import Modal from 'antd/lib/modal/Modal'
+import { NftPostPayModalConfig } from 'models/nftRewardTier'
+import { withHttps } from 'utils/externalLink'
+import { windowOpen } from 'utils/windowUtils'
+
+export function NftPostPayModal({
+  visible,
+  config,
+  onClose,
+}: {
+  visible: boolean
+  config: NftPostPayModalConfig
+  onClose: VoidFunction
+}) {
+  const onOk = () => {
+    if (config.ctaLink) {
+      windowOpen(config.ctaLink)
+    } else {
+      onClose()
+    }
+  }
+  return (
+    <Modal
+      visible={visible}
+      onCancel={onClose}
+      onOk={onOk}
+      title={t`NFT mint confirmed`}
+      okButtonProps={{
+        href: withHttps(config.ctaLink),
+        target: '_blank',
```

Comment:
**suggestion**

```suggestion
        target: '_blank',
        rel='noopener noreferrer'
```

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/